execute_query.lua

```diff
@@ -13,9 +13,6 @@ local explain = false
 local port = 3301
 local mem_size = 10 * 1024^3
 
--- FIXME -- Q13 is 4h50mins long, Q17 - 48mins, and Q20 - 1h11mins
-local excluded_tests = {13, 17, 20}
```

Comment:
From time to time we will need to disable particular tests. Could you please not remove this code entirely, but rather leave empty table?